modules/distribution/src/main/config/fluo-env.sh

```diff
@@ -11,11 +11,41 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 
-# Sets HADOOP_PREFIX if it is not already set.  Please modify the
-# export statement to use the correct directory.  Remove the test
-# statement to override any previously set environment.
+## Before fluo-env.sh is loaded, these environment variables are set and can be used in this file:
 
-test -z "$HADOOP_PREFIX" && export HADOOP_PREFIX=/path/to/hadoop
+# cmd - Command that is being called such as oracle, worker, etc.
+# app - Fluo application name 
+# basedir - Root of Fluo installation
+# conf - Directory containing Fluo configuration
+# lib - Directory containing Fluo libraries
+
+############################
+# Variables that must be set
+############################
+
+## Fluo logs directory. Referenced by logger config.
+export FLUO_LOG_DIR="${FLUO_LOG_DIR:-${basedir}/logs}"
+## Hadoop installation
+export HADOOP_PREFIX="${HADOOP_PREFIX:-/path/to/hadoop}"
+## Fluo connection properties
+export FLUO_CONN_PROPS="${FLUO_CONN_PROPS:-${conf}/fluo-conn.properties}"
+## Fluo log4j configuration
+export FLUO_LOG4J_CONFIG="${FLUO_LOG4J_CONFIG:-${conf}/log4j.properties}"
+
+##################################################################
+# Build JAVA_OPTS variable. Defaults below work but can be edited.
+##################################################################
+
+export FLUO_LOG_ID="${cmd}_$(hostname)_$(date +%s)"
+
+JAVA_OPTS=("-Dlog4j.configuration=file:${FLUO_LOG4J_CONFIG}"
```

Comment:
It seems like these JAVA_OPTS are only used for services?  If so, I think the name should changes.  At first I thought maybe all of the commands would use this (like `fluo init`).  Also, if all of the logging stuff is only used for services it would be nice to group all of the service config into a section (delimited  by the `############`)